src/snapred/backend/service/MetadataLookupService.py

```diff
@@ -0,0 +1,44 @@
+from mantid.kernel import IntArrayProperty
+
+from snapred.backend.data.DataFactoryService import DataFactoryService
+from snapred.backend.log.logger import snapredLogger
+from snapred.backend.service.Service import Service
+from snapred.meta.Config import Config
+from snapred.meta.decorators.FromString import FromString
+from snapred.meta.decorators.Singleton import Singleton
+from snapred.meta.validator.RunNumberValidator import RunNumberValidator
+
+logger = snapredLogger.getLogger(__name__)
+
+
+@Singleton
+class MetadataLookupService(Service):
+    dataFactoryService = "DataFactoryService"
+
+    # register the service in ServiceFactory please!
+    def __init__(self):
+        super().__init__()
+        self.dataFactoryService = DataFactoryService()
+        self.registerPath("", self.verifyMultipleRuns)
+        return
+
+    @staticmethod
+    def name():
+        return "verifiedRuns"
+
+    @FromString
+    def verifyMultipleRuns(self, runs: str):
+        maxRuns = Config["instrument.maxNumberOfRuns"]
+        try:
+            allRuns = IntArrayProperty(name="RunList", values=runs).value
+        except Exception as e:
+            raise e
```

Comment:
Is this try/catch necessary here?

Reply:
I guess I thought it was, but it seems like IntArrayProperty will throw an exception internally if the input is bad